lib/yuriita/query/input.rb

```diff
@@ -16,7 +16,11 @@ def ==(other)
       alias_method :eql?, :==
 
       def to_s
-        "#{qualifier}:#{term}"
+        if term.match(" ")
```

Comment:
I think this will return a `<MatchData>` object. Would using [`.match?`](http://ruby-doc.org/core-2.7.1/String.html#//dash_ref_method%252Di%252Dmatch%252D3F/Method/match?/0) be more appropriate?

```suggestion
        if term.match?(" ")
```

Reply:
good catch. thanks!